developers/ide/vscode.md

```diff
@@ -15,7 +15,13 @@ The following steps will only need to be done once to setup both VSCode and your
    
 2. Clone the addons (https://github.com/openhab/openhab-addons.git or preferably your own fork) to %BASE%\openhab-addons
    
-3. If you want to setup openHAB code formatting guidelines, add the following to the VSCode settings:
+3. On Windows, VSCode should be configured to use powershell instead of the classic command line. If not configured already, add the following to the VSCode settings:
+         
+     ```"terminal.integrated.automationShell.windows": "C:\\Windows\\sysnative\\WindowsPowerShell\\v1.0\\powershell.exe"```
+
+     Either globally define the options via the global ```settings.json```.  You can also define them local to a specific bundle by putting those lines in the ```.vscode/settings.json``` file in the bundle (similar to ```tasks.json```/```launch.json``` below).
+
+4. If you want to setup openHAB code formatting guidelines, add the following to the VSCode settings:
```

Comment:
```suggestion
1. If you want to setup openHAB code formatting guidelines, add the following to the VSCode settings:
```

Reply:
Why do you want to change all numbers to 1.?

Reply:
Those are just markes for an "ordered list".
Our website build mechanism will take care of the correct numbering during the build.

So you simply don't have to worry if you are working on step 1 step 7, step 3 or whatever step is done in an ordered list currently.

I am trying to generalise all ordered lists in this style on long term.

Reply:
Thanks for the explanation.